front/src/modules/timesStops/helpers/scheduleData.ts

```diff
@@ -30,6 +32,6 @@ export function formatScheduleData(scheduleData: ComputedScheduleEntry) {
   return {
     arrival: scheduleData.arrival ? secToHoursString(scheduleData.arrival, true) : '',
     departure: scheduleData.departure ? secToHoursString(scheduleData.departure, true) : '',
-    stopFor: scheduleData.stopFor ? String(scheduleData.stopFor) : '',
+    stopFor: scheduleData.stopFor !== null ? String(scheduleData.stopFor) : '',
```

Comment:
This solution solves the bug of 0s not being displayed in the output table, but we still have the problem we discussed together.
1. On the input table enter a value in one of the stop for.
2. Delete this value.
3. Add your train.
4. In the output table we find 0s in the stop for when it shouldn't be